From 48beadca0b214594490f4237b17f2c95cb24ea04 Mon Sep 17 00:00:00 2001
From: Nikolaus Schulz <nikolaus.schulz@avionic-design.de>
Date: Mon, 2 Feb 2015 18:57:28 +0100
Subject: [PATCH 1/3] Fix cross compilation by making the build honor CC,
 CFLAGS and LDFLAGS

The build system for this package is rather crude, and wants users to
hand-edit the files conf-cc and conf-ld in order to customize the build.
Change these to expand $CC and auxiliary variables.

Signed-off-by: Nikolaus Schulz <nikolaus.schulz@avionic-design.de>
---
 conf-cc | 2 +-
 conf-ld | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/conf-cc b/conf-cc
index b99d03d..ca978ab 100644
--- a/conf-cc
+++ b/conf-cc
@@ -1,3 +1,3 @@
-cc -O2 -DPOSIX -D_POSIX_C_SOURCE=200809L -D_FILE_OFFSET_BITS=64 -DTEST_NARROW_WRITES -c
+${CC-cc} $CPPFLAGS $CFLAGS -O2 -DPOSIX -D_POSIX_C_SOURCE=200809L -D_FILE_OFFSET_BITS=64 -DTEST_NARROW_WRITES
 
 This will be used to compile .c files.
diff --git a/conf-ld b/conf-ld
index a9e796a..cdc0aba 100644
--- a/conf-ld
+++ b/conf-ld
@@ -1,3 +1,3 @@
-cc -s
+${CC-cc} $LDFLAGS
 
 This will be used to link .o files into an executable.
-- 
2.1.4

